packages/gasket-plugin-intl/lib/build-manifest.js

```diff
@@ -50,6 +50,10 @@ module.exports = async function buildManifest(gasket, options = {}) {
       const keyName = importName
         .replace(/\.json$/, '')
         .replace('./', '');
+      if (gasket.config.intl.experimentalImportAttributes) {
```

Comment:
y'all should create a top level experimental/future flag which uses import asserts. 

import as/from are executed at build time. createRequire is  executed at runtime which makes it harder to run gasket in serverless environments